src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -68,6 +68,11 @@ public function load(array $configs, ContainerBuilder $container)
 
         if ($container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
             $loader->load('security_debug.xml');
+
+            if ($container->hasDefinition('security.access.decision_manager') && $container->hasDefinition('debug.security.access.decision_manager')) {
```

Comment:
This is not necessary, the last loaded `autowiring_types` config will override previous ones. You can just add `<autowiring-type>Symfony\Component\Security\Core\Authorization\AccessDecisionManagerInterface</autowiring-type>` to the `debug.security.access.decision_manager` and it should do the trick without having to touch the extension.